client/gutenberg-blocks/README.md

```diff
@@ -0,0 +1,16 @@
+Gutenberg Blocks
+====
+
+[Gutenberg](https://wordpress.org/gutenberg/) is the new publishing experience for WordPress.
+
+The Gutenberg editor uses 
+[blocks](https://github.com/WordPress/gutenberg/blob/master/blocks/README.md) to create all types
+of content. A Gutenberg block is the abstract term used to describe units of markup that, 
+composed together, form the content or layout of a webpage.
```

Comment:
This one is a little hard to condense in a single sentence, but maybe we can link out to additional sources like https://wordpress.org/gutenberg/handbook/language/